deployments/hammerbox/environment.py

```diff
@@ -396,6 +396,325 @@ def mkdict(previous_catalog: dict[str, str],
+    mksrc('datarepo-beef6734', 'ANVIL_CMH_GAFK_MGI_20240304_ANV5_202403071559', 1609),
+    mksrc('datarepo-8599b1fb', 'ANVIL_CMH_GAFK_PacBio_methyl_tagged_20240311_ANV5_202403121402', 793),
+    mksrc('datarepo-94f58e6c', 'ANVIL_CMH_GAFK_SCATAC_20221107_ANV5_202402290954', 322),
+    mksrc('datarepo-5447de30', 'ANVIL_CMH_GAFK_WGBS_20230327_ANV5_202402062120', 1475),
+    mksrc('datarepo-db73a316', 'ANVIL_CMH_GAFK_WGS_20240113_ANV5_202402062123', 23848),
+    mksrc('datarepo-5227851b', 'ANVIL_CSER_ClinSeq_GRU_20240401_ANV5_202404081541', 23),
+    mksrc('datarepo-1a706b0c', 'ANVIL_GTEx_Somatic_WGS_20240116_ANV5_202401170147', 708, pop),
+    mksrc('datarepo-6cff3240', 'ANVIL_HPRC_20240401_ANV5_202404122012', 63201),
+    mksrc('datarepo-52aaeb56', 'ANVIL_MAS_ISO_seq_20240113_ANV5_202404081635', 205),
+    mksrc('datarepo-8a98bcb4', 'ANVIL_NIMH_Broad_ConvNeuro_McCarroll_Nehme_Levy_CIRM_DS_Village_20240405_ANV5_202404081511', 495), # noqa E501
+    mksrc('datarepo-c02a5efb', 'ANVIL_NIMH_Broad_ConvergentNeuro_McCarroll_Eggan_CIRM_GRU_VillageData_20230109_ANV5_202402292203', 1357), # noqa E501
+    mksrc('datarepo-817f27aa', 'ANVIL_NIMH_Broad_ConvergentNeuro_McCarroll_Eggan_CIRM_GRU_WGS_20240206_ANV5_202402081755', 678), # noqa E501
+    mksrc('datarepo-ddc1d72b', 'ANVIL_NIMH_Broad_ConvergentNeuro_McCarroll_Eggan_Finkel_SMA_DS_WGS_20230109_ANV5_202402292209', 3), # noqa E501
+    mksrc('datarepo-14f5afa3', 'ANVIL_NIMH_Broad_WGSPD1_McCarroll_Braff_DS_10XLRGenomes_20221115_ANV5_202310101713', 561, pop), # noqa E501
+    mksrc('datarepo-69e4bc19', 'ANVIL_NIMH_Broad_WGSPD1_McCarroll_COGS_DS_WGS_20240113_ANV5_202401152215', 611),
+    mksrc('datarepo-da595e23', 'ANVIL_NIMH_Broad_WGSPD1_McCarroll_Escamilla_DS_WGS_20240112_ANV5_202401141541', 340),
+    mksrc('datarepo-55fb8d04', 'ANVIL_NIMH_Broad_WGSPD1_McCarroll_Light_DS_WGS_20240102_ANV5_202402070117', 60),
+    mksrc('datarepo-94091a22', 'ANVIL_NIMH_Broad_WGSPD1_McCarroll_Pato_GRU_10XLRGenomes_20230331_ANV5_202310101715', 368, pop), # noqa E501
+    mksrc('datarepo-df20901c', 'ANVIL_NIMH_Broad_WGSPD_1_McCarroll_Braff_DS_WGS_20240304_ANV5_202403071610', 213),
+    mksrc('datarepo-75e17b99', 'ANVIL_NIMH_CIRM_FCDI_ConvergentNeuro_McCarroll_Eggan_GRU_Arrays_20230109_ANV5_202402292215', 6510), # noqa E501
+    mksrc('datarepo-25790186', 'ANVIL_PAGE_BioMe_GRU_WGS_20221128_ANV5_202403040429', 308, pop),
+    mksrc('datarepo-b371989b', 'ANVIL_PAGE_MEC_GRU_WGS_20230131_ANV5_202403040437', 70, pop),
+    mksrc('datarepo-4a4eec27', 'ANVIL_PAGE_SoL_HMB_WGS_20221220_ANV5_202403040445', 234, pop),
+    mksrc('datarepo-a1f917db', 'ANVIL_PAGE_Stanford_Global_Reference_Panel_GRU_WGS_20221128_ANV5_202403040453', 78, pop), # noqa E501
+    mksrc('datarepo-6264931f', 'ANVIL_PAGE_WHI_HMB_IRB_WGS_20221019_ANV5_202403040500', 235, pop),
+    # FIXME: Update subgraphs count script fails for ANVIL_T2T_CHRY dataset
+    #        https://github.com/DataBiosphere/azul/issues/6206
```

Comment:
This FIXME is out of context since it doesn't apply to the code immediately below it. I believe we discussed this as well in PL.

I think it's a good idea to have a commit referring #6206, but that commit can't add a FIXME comment and its title should indicate that it constitutes a workaround.